plugins/kubernetes/app/views/kubernetes/roles/example.html.erb

```diff
@@ -0,0 +1,69 @@
+<%= render 'projects/header', project: @project, tab: 'kubernetes' %>
+
+<section class="tabs kubernetes-section clearfix" xmlns="http://www.w3.org/1999/html">
+  <%= render 'samson_kubernetes/navigation' %>
+
+  <h2>Example kubernetes config</h2>
+  A web-server would look like this:
+
+  <% project_label = @project.permalink.tr('_', '-') %>
+  <pre>
+# kubernetes/app-server.yml
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: <%= project_label %>-app-server
+  labels:
+    project: <%= project_label %>
+    role: app-server
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      project: <%= project_label %>
+      role: app-server
+  strategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      name: <%= project_label %>-app-server
+      labels:
+        project: <%= project_label %>
+        role: app-server
```

Comment:
Could you add some example annotations here for secrets, with a comment on how they can be used?